arrows/vxl/convert_image.cxx

```diff
@@ -46,7 +46,7 @@ scale_image( vil_image_view< InType > const& src,
 
   vil_transform(
     src, dst,
-    [ max_input_value, max_output_value, dp_scale ]( InType pixel ){
+    [ max_input_value, dp_scale ]( InType pixel ){
```

Comment:
What was the reason for removing this input? It appears to be a breaking change for the Windows build, with the following error:
```
C:\workspace\KwiverWindowsPR\arrows\vxl\convert_image.cxx(57):
error C3493: 'max_output_value' cannot be implicitly captured because
 no default capture mode has been specified
[C:\workspace\KwiverWindowsPR\build\arrows\vxl\kwiver_algo_vxl.vcxproj]
```

Reply:
Augh! See 5fa46f9c33859714eaafcfc1e516d2fd6edd9d3c:
> Don't capture the maximum output value in the lambda used to transform and scale images in the vxl_convert_image algorithm. Doing so is redundant, since constants from the enclosing context do not need to be explicitly captured, and doing so may result in a compiler warning.

Neither GCC nor clang needs an explicit capture, and clang [*complains*](https://godbolt.org/z/Kr1M79) about it being captured. Apparently MSVC (all available versions) disagrees. However, [this SO question](https://stackoverflow.com/questions/33873788) suggests that MSVC is wrong, and indeed, so does [the standard](http://open-std.org/jtc1/sc22/wg21/docs/papers/2012/n3337.pdf) (n.b. the example at `[expr.prim.lambda]¶12`).

Sigh. I think I'm going to replace this with a `[=]` and a nasty commit message about M$'s POG compiler.